Proiect-VS/assets/css/new-evaluation.less

```diff
@@ -0,0 +1,162 @@
+body {
+    position: relative;
+    margin: auto;				
+}
+
+.skin_body {
+    background-color: #f7f7f7;
+    font-family: Arial;
+}
+
+header {
+    height: 70px;
+    border-bottom: 1px solid grey;
+    font-size: 25px;
+}
+        
+footer {
+    height: 70px; 
+    line-height: 70px;
+    vertical-align: middle;
+    text-align: right;
+    padding-right: 50px;			
+    margin-top: 10px;
+    position: absolute;
+    left: 0;
+    right: 0;
+    float: left;
+}	
+
+.skin_footer {
+    background-color: #bcbcaf;
+    color: #686865;	
+}
+
+.logo {
+    width: auto;
+    height: 50px;
+    line-height: 70px;
+    vertical-align: middle;
+}
+
+.menu_button {
+    vertical-align: middle;
+    line-height: 70px;
+    margin-left: 100px;
+    margin-right: 15px;
+}
+
+.current_menu_button {
+    border-bottom: 9px solid #21bcff; 
+    padding-bottom: 12px;
+}
+
+a:link, a:visited {
+    text-decoration: none;
+    color: black;
+}
+
+.evaluation_form {
+    padding-top: 55px;
+    margin: 0 auto ;
+    width: 1000px;
+    height: 40px;
+}
+
+.evaluation_form_header {
+    border: none;
+    border-bottom: 3px solid #f7f7f7;
+    height: 30px;
+    width: 300px;
+    margin: 7px;
+}
+
+.evaluation_form_table {
+    border-collapse: collapse;
+    width: 1000px;
+    margin: 20.750px 0px;
```

Comment:
Be mindful when using sub-unit measurements, since they might not get properly displayed on all devices / browsers / environments. Unless there's good enough reason why you might want this level of accuracy, I'd most often than not recommend avoiding the complications.